nighthawk/include/nighthawk/client/options.h

```diff
@@ -27,6 +27,8 @@ class Options {
   virtual bool h2() const PURE;
   virtual std::string concurrency() const PURE;
   virtual std::string verbosity() const PURE;
+  virtual std::string output_format() const PURE;
```

Comment:
Nit: `outputFormat`

Reply:
renamed (+requests_per_second->requestsPerSecond for consistency)